### PR TITLE
Cleaned up documentation.

### PR DESCRIPTION
**Pull request by jeffrey-opdam**
Thank you for your pull request!
By completing this pull request, you agree to the [Contributing License Agreement](https://github.com/ALM-Rangers/Team-Services-Bot/blob/master/.github/CLA.md).

Fixes # .

Changes proposed in this pull request:  
- Make documentation consistent.
- Removed some documentation from the test projects. 
- 

@ALM-Rangers/Team-Services-Bot

### CODE REVIEW

File: src/Team-Services-Bot.AcceptanceTests/Team-Services-Bot.AcceptanceTests.csproj [hunk 1]
```diff
@@ -34,6 +34,7 @@
     <DocumentationFile>bin\Debug\Vsar.TSBot.AcceptanceTests.xml</DocumentationFile>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <CodeAnalysisIgnoreGeneratedCode>true</CodeAnalysisIgnoreGeneratedCode>
+    <NoWarn>1591</NoWarn>
```
Comment (Hussar77) [inline src/Team-Services-Bot.AcceptanceTests/Team-Services-Bot.AcceptanceTests.csproj]: I prefer disabling such things in the code (#pragma warning disable) as the test class library might have helper classes that require XML doc.

Comment (Hussar77) [inline src/Team-Services-Bot.AcceptanceTests/Team-Services-Bot.AcceptanceTests.csproj]: Also, that option should be applied for all configurations.

File: src/Team-Services-Bot.Api.UnitTests/Team-Services-Bot.Api.UnitTests.csproj [hunk 2]
```diff
@@ -33,6 +33,7 @@
     <CodeAnalysisRuleSet>..\Common\TestCodeRules.ruleset</CodeAnalysisRuleSet>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <DocumentationFile>bin\Debug\Vsar.TSBot.UnitTests.xml</DocumentationFile>
+    <NoWarn>1591</NoWarn>
```
Comment (Hussar77) [inline src/Team-Services-Bot.Api.UnitTests/Team-Services-Bot.Api.UnitTests.csproj]: Same issue - should be disabled for release configuration as well. Preferable this should be done in the code.

Comment (jeffrey-opdam) [inline src/Team-Services-Bot.Api.UnitTests/Team-Services-Bot.Api.UnitTests.csproj]: I have updated so it is for all configurations

File: src/Team-Services-Bot.AcceptanceTests/Team-Services-Bot.AcceptanceTests.csproj (hunk 1 above)
Comment (jeffrey-opdam) [inline src/Team-Services-Bot.AcceptanceTests/Team-Services-Bot.AcceptanceTests.csproj]: That would be a lot of pragma's

**Pull request merged**